routes/curations.js

```diff
@@ -69,6 +70,13 @@ async function updateCurations(request, response) {
   })
 }
 
+router.post('/sync', permissionsCheck('curate'), asyncMiddleware(syncAllContributions))
```

Comment:
make sure this setting is in all the environments. 

`AUTH_CURATION_TEAM=curation-dev|curation-prod`